runtime/drivers/object_store.go

```diff
@@ -42,6 +42,6 @@ type ObjectStoreModelOutputProperties struct {
 }
 
 type ObjectStoreModelResultProperties struct {
-	Path   string     `mapstructure:"path"`
-	Format FileFormat `mapstructure:"format"`
+	Path   string `mapstructure:"path"`
+	Format string `mapstructure:"format"`
```

Comment:
Why this change?

Reply:
When we are storing the results in catalog here : https://github.com/rilldata/rill/blob/b970338f4df225c7fc3d1acb86c27872ae52fe5c/runtime/reconcilers/model.go#L523-L524
it is not able to handle `FileFormat` type
It needs go default types only.

Reply:
I think we do not hit this case right now but we will hit this case if we just have a model which has `snowflake` as input and `s3` as output.

Reply:
Got it